internal/token.go

```diff
@@ -1,9 +1,20 @@
 // Package internal contains utilities internal to the weasel packages.
 package internal
 
-import "golang.org/x/oauth2/google"
+import (
+	"golang.org/x/oauth2/google"
+	"golang.org/x/net/context"
```

Comment:
I believe x/net/context is no more and it's always just "context", a package in stdlib.

Reply:
fixed